### PR TITLE
`SortingAnalyzer` in "how to" analyze neuropixels

### PR DESCRIPTION
**Pull request by samuelgarcia**

### CODE REVIEW

**Comment by zm711** [comment]
@samuelgarcia, I was just briefly looking at this and it looks like you reintroduced some things that have been changed since the original version of this was made.

For example Pierre switched `local_radius_um` -> `radius_um`. I also corrected a bunch of typos but your new version reintroduces the typos. Not sure how you want to deal with this though... But if you commit the current version the tutorial will have coding mistakes for the current api.

**Comment by samuelgarcia** [comment]
Oups. I forgot about this radius.
I guess this is because I changed the py and regenrated the rst.
Maybe you only changed the rst no ?
I will fix this.

File: examples/how_to/analyse_neuropixels.py [hunk 1]
```diff
@@ -78,39 +78,33 @@
 
 #
 #
-# The preprocessing steps can be interactively explored with the ipywidgets interactive plotter
+# Interactive explore the preprocess steps could de done with this with the ipywydgets interactive ploter
 #
 # ```python
 # # %matplotlib widget
-# si.plot_traces({'filter':rec1, 'cmr': rec4}, backend='ipywidgets')
+# si.plot_timeseries({'filter':rec1, 'cmr': rec4}, backend='ipywidgets')
```
Comment (zm711) [inline examples/how_to/analyse_neuropixels.py:85]: This needs to be fixed too!

File: examples/how_to/analyse_neuropixels.py [hunk 2]
```diff
@@ -174,137 +168,148 @@
...
+job_kwargs = dict(n_jobs=40, chunk_duration='1s', progress_bar=True)
+peaks = detect_peaks(rec,  method='locally_exclusive', noise_levels=noise_levels_int16, 
+                     detect_threshold=5, local_radius_um=50., **job_kwargs)
 peaks
 
 # +
 from spikeinterface.sortingcomponents.peak_localization import localize_peaks
 
-peak_locations = localize_peaks(rec, peaks, method="center_of_mass", radius_um=50.0, **job_kwargs)
+peak_locations = localize_peaks(rec, peaks, method='center_of_mass', local_radius_um=50., **job_kwargs)
```
Comment (zm711) [inline examples/how_to/analyse_neuropixels.py:179]: here

**Comment by zm711** [comment]
Actually the fixes are in the .py file for typos too. Don't know what happened. Either way not a biggie.

File: examples/how_to/analyse_neuropixels.py (hunk 1 above)
Comment (samuelgarcia) [reply examples/how_to/analyse_neuropixels.py]: done

File: examples/how_to/analyse_neuropixels.py (hunk 2 above)
Comment (samuelgarcia) [reply examples/how_to/analyse_neuropixels.py]: done

**Pull request merged**